openmetadata-ui/src/main/resources/ui/src/components/Domain/DomainTabs/DocumentationTab/DocumentationTab.component.tsx

```diff
@@ -0,0 +1,319 @@
+      owner: newOwner,
+    };
+    onUpdate(updatedData);
+  };
+
+  const handleExpertsUpdate = (data: Array<EntityReference>) => {
+    if (!isEqual(data, domain.experts)) {
+      let updatedDomain = cloneDeep(domain);
+      const oldExperts = data.filter((d) => includes(domain.experts, d));
+      const newExperts = data
+        .filter((d) => !includes(domain.experts, d))
+        .map((d) => ({ id: d.id, type: d.type }));
+      updatedDomain = {
+        ...updatedDomain,
+        experts: [...oldExperts, ...newExperts],
+      };
+      onUpdate(updatedDomain);
+    }
+  };
+
+  const handleDomainTypeUpdate = async (domainType: string) => {
+    let updatedDomain = cloneDeep(domain);
+    updatedDomain = {
+      ...updatedDomain,
+      domainType: domainType as DomainType,
+    };
+    await onUpdate(updatedDomain);
+    setEditDomainType(false);
+  };
+
+  return (
+    <Row>
+      <Col className="border-right p-md domain-content-container" span={18}>
+        <DescriptionV1
+          description={domain.description}
+          entityName={getEntityName(domain)}
+          entityType={EntityType.DOMAIN}
+          hasEditAccess={editDescriptionPermission}
+          isEdit={isDescriptionEditable}
+          showCommentsIcon={false}
+          onCancel={() => setIsDescriptionEditable(false)}
+          onDescriptionEdit={() => setIsDescriptionEditable(true)}
+          onDescriptionUpdate={onDescriptionUpdate}
+        />
+      </Col>
+      <Col className="p-md" span={6}>
+        <Row gutter={[0, 40]}>
+          <Col data-testid="domain-owner-name" span="24">
+            <div className="d-flex items-center m-b-xss">
+              <Typography.Text className="right-panel-label">
+                {t('label.owner')}
+              </Typography.Text>
+              {editOwnerPermission && domain.owner && (
+                <UserTeamSelectableList
+                  hasPermission
+                  owner={domain.owner}
+                  onUpdate={handleUpdatedOwner}>
+                  <Button
+                    className="cursor-pointer flex-center m-l-xss"
+                    data-testid="edit-owner"
+                    icon={<EditIcon color={DE_ACTIVE_COLOR} width="14px" />}
+                    size="small"
+                    type="text"
+                  />
+                </UserTeamSelectableList>
+              )}
+            </div>
+
+            <Space className="m-r-xss" size={4}>
+              {getOwner(domain.owner)}
+            </Space>
+
+            {!domain.owner && editOwnerPermission && (
+              <UserTeamSelectableList
+                hasPermission
+                owner={domain.owner}
+                onUpdate={handleUpdatedOwner}>
+                <TagButton
+                  className="tw-text-primary cursor-pointer"
+                  icon={<PlusIcon height={16} name="plus" width={16} />}
+                  label={t('label.add')}
+                  tooltip=""
+                />
```

Comment:
Can we use OwnerLabel here for consistancy?

Reply:
Will update in a separate PR